src/oop/oop2.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, num_wheels = 4):
         #self.num_wheels = num_wheels
```

Comment:
You might be able to set the default arg of num_wheels to 2 here, or just skip it entirely. Not 100% sure on that.